CSS/style.css

```diff
@@ -0,0 +1,184 @@
+@import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+    margin: 0%;
+    padding: 0%;
+    background-color: #FFF;
+}
+
+.My_logo {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 20px;
+    color: #6070ff;
+}
+
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    padding: 26px 164px 26px 155px;
+    align-items: center;
+    padding-bottom: 4%;
+    background-color: #FFF;
+}
```

Comment:
- [ ] I have conned your repo to check the changes and avoid errors, but it shows some differences with the Figma template 👇🏽 

Please review it with the 375 media for the mobile version, as requested by previous code reviewers your design should match the Figma template, please 🙏🏽 correct the following things:

- The space between the logo and the hamburger button.
- The number and order of social icons:
   - Twitter ✔️ 
   - LinkedIn ✔️ 
   - Medium ❌ 
   - Github ✔️ 
   - Angelist (Wellfound) ❌ 
   - ~~Facebook~~

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219679719-a9b687bd-6fab-4748-98d7-af48cb90eed6.png) | ![image](https://user-images.githubusercontent.com/58642949/219679855-f46af144-3429-4caa-838e-b9a8b96741b2.png) |